tests/command-data.test.js

```
@@ -0,0 +1,91 @@
+ * rather than loading the actual module
+ */
+
+describe('command-data.js', () => {
+  // These tests verify the expected handler behaviors
+  // without loading the actual module
+  
+  describe('Command Navigation Handlers', () => {
+    it('should handle navigation to home page', () => {
+      // Mock window.location
+      const originalLocation = window.location;
+      delete window.location;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)